cmd/vcert/utils.go

```diff
@@ -44,6 +44,8 @@ const (
 	/* #nosec */
 	vCertApiKey      = "VCERT_APIKEY"
 	vCertTrustBundle = "VCERT_TRUST_BUNDLE"
+
+	JKSFormat = "jks"
```

Comment:
shouldn't this be a constant?

Reply:
ahhh... but it is in a const block

Reply:
My bad, it is indeed in a constant block. All good